plugins/examples/__init__.py

```diff
@@ -521,6 +521,57 @@ def execute(self, ctx):
         )
         return {}
 
+class ResolveParamsExample(foo.Operator):
+    @property
+    def config(self):
+        return foo.OperatorConfig(
+            name="example_resolve_params",
+            label="Examples: Resolve Params",
+            dynamic=True,
+        )
+
+    def resolve_params(self, ctx, previous):
+        print(previous)
+        return {"msg": "Hello World!"}
+    
+    def resolve_input(self, ctx):
+        inputs = types.Object()
+        inputs.str("msg", label="Message")
+        return types.Property(inputs)
+    
+    def execute(self, ctx):
+        return {"msg": ctx.params["msg"]}
+    
+class FileDropExample(foo.Operator):
+    @property
+    def config(self):
+        return foo.OperatorConfig(
+            name="example_file_drop",
+            label="Examples: File Drop",
+            dynamic=True,
+        )
+
+    def resolve_input(self, ctx):
+        inputs = types.Object()
+        ten_mb = 1024*1024*10
+        file_drop = types.FileView(label="File Drop", max_size=ten_mb, max_size_error_message="File too large")
+        inputs.obj("file", view=file_drop)
+        return types.Property(inputs)
+
+    def execute(self, ctx):
+        return {"file": ctx.params['file']}
+
+    def resolve_output(self, ctx):
+        outputs = types.Object()
+        file_obj = types.Object()
```

Comment:
Once mapping is added, we should probably use `types.UploadedFile()` here

Reply:
Meant to comment this for resolve_input. However, object would also work fine. I wonder if it would be useful to support output view.